### PR TITLE
Add styles to the bookstore app

### PR DESCRIPTION
**Pull request by ezraermy**
This pull request:
- Creates styles for the bookstore app according to [Zeplin project](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) bookstore CMS.

## Other essential requirements:
- Use Html/CSS
- Use JavaScript ES6
- reactJS
- Use external API
- Make sure linter for free of errors.
- Using GitHub flow.
- Documented in a professional way.

### CODE REVIEW

**Review by brainconnect93 — CHANGES_REQUESTED** [review]
## STATUS: Changes Required :recycle:

### Hi @ezraermy, 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️  
- Styling of the bookstore CMS project ✔️ 
- Zeplin Design template used for the design ✔️ 
- There is good use of Git flow ✔️ 

### Changes Required

- [ ] Kindly work on the changes i requested from you via the video demonstration below 👇 

https://user-images.githubusercontent.com/78634841/231835351-9f8b6115-482d-4eba-88b7-e803954d6759.mp4



- [x]  _Please check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @brainconnect93 ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
@brainconnect93 That's an excellent video explanation. I very much liked it and Thank you for your detailed feedback. I will make sure everything is inorder and working based on your suggestion.

**Comment by ezraermy** [comment]
@brainconnect93  You can see from the picture, the form is perfectly adding and removing the books. I'vnt saw any problems with the form. Do you have any ideas before I make a second pull request?

**Comment by ezraermy** [comment]
Screenshot from 2023-04-13 20-45-18
@brainconnect93 As you can see from the pic above, the form works perfectly fine. I'vnt saw problems with adding and removing a book. Is anything you can suggest, I can go through and see?

File: src/components/Form.js [hunk 1]
```diff
@@ -30,27 +32,28 @@ const Form = () => {
...
+          placeholder="Book Author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+          className="author-input"
+        />
+        <button type="button" onClick={handleAddBook} className="add-btn">
+          Add Book
+        </button>
+      </div>
+      {message && <p className="error">{message}</p>}
```
Comment (Zuhaib042) [inline src/components/Form.js]: - [ ] As the previous reviewer suggested that whenever a user tries to add a book, the `Loading...` appears but the book doesn't get added and it doesn't show any books from API. I have cloned this project on my machine I ran the server on the latest commit. 
One thing I noticed is that there is an error when you run the server and another error when you try to add a book. Kindly consider fixing these errors.
your project 

https://user-images.githubusercontent.com/96683843/231849831-13f04496-f5b1-453c-a40c-91ade7935e69.mov


errors in console
re

File: src/components/styles/Form.css [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+.form-title {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 2.813rem 32.75rem 0.188rem 3.5rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```
Comment (Zuhaib042) [inline src/components/styles/Form.css]: - [OPTIONAL] As `normal` is the default values and I can see that you are writing similar lines in all of your CSS files. Actually, you have copied it from the zeplin. there is no problem is that but you should remove these lines as they are redundant and against DRY principle.

File: src/components/styles/Form.css [hunk 3]
```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: 'Montserrat';
+  src: url('https://fonts.googleapis.com/css?family=Montserrat') format('truetype');
+}
+
+.input-form {
+  display: flex;
+  flex-direction: row;
```
Comment (Zuhaib042) [inline src/components/styles/Form.css:8]: - [OPTIONAL] One golden tip in styling is that we can create a class with any styling that is repeating and then give that class to any element we want. for example I can write
```
.flex {
display: flex;
}
```
and then give `flex` class to any container I want. you can apply this rule to save a lot of lines in your CSS.

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @ezraermy 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (ezraermy) [reply src/components/Form.js]: @Zuhaib042 Thank you very much for the feedback. I think I know the reason. This is based on the suggestion made by the yesterday's reviewer. I just hid my key and used .env instead. I will fix that and make it available.

**Review by GedeonTS — APPROVED** [review]
# STATUS: APPROVED :tada: :clap: :champagne: 
Hi @ezraermy ,

You are amazing :smiley: :clap: !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

 

## Highlights :clap: :tophat: 
- No linter errors :+1: 
- The books are added and removed properly :+1: 
- The app is styled following the provided design :+1: 
## Optional suggestions :small_airplane: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ezraermy** [comment]
@GedeonTS Thanks a lot for taking the time and go through my projects.

**Pull request merged**